wondrous-app/components/Settings/styles.tsx

```diff
@@ -227,6 +227,32 @@ export const SettingsHeaderText = styled(Typography)`
   }
 `;
 
+export const SettingsHeaderInviteButton = styled.button`
+  display: flex;
```

Comment:
Can you use this common button `components/Button/index.tsx` or create a new one (here `components/Button/index.tsx`)  to prevent duplicated code?

Reply:
Have removed duplicate code for Invite buttons inside the modal. I believe this button should be the way it is, it has different styles and is part of a seperate component altogether. Let me know if you feel otherwise.